vic/vic_run/src/compute_derived_lake_dimensions.c

```diff
@@ -85,19 +85,19 @@ compute_derived_lake_dimensions(lake_var_struct *lake,
         }
         status = get_sarea(lake_con, depth, &(lake->surface[k]));
         if (status < 0) {
-            log_err("Error in get_sarea: record = %d, depth = %f, "
+            log_err("Error: record = %d, depth = %f, "
                     "sarea = %e", 0, depth, lake->surface[k]);
         }
     }
 
     lake->sarea = lake->surface[0];
     status = get_volume(lake_con, lake->ldepth, &tmp_volume);
     if (status < 0) {
-        log_err("Error in get_volume: record = %d, depth = %f, "
+        log_err("Error: record = %d, depth = %f, "
                 "volume = %e", 0, depth, tmp_volume);
     }
     else if (status > 0) {
-        log_err("Warning in get_volume: lake depth exceeds maximum; "
+        log_err("Warning: lake depth exceeds maximum; "
```

Comment:
these two are confusing.  I would just drop the Error/Warning portions of these strings.